examples/axum/templates/page.rs.html

```diff
@@ -0,0 +1,36 @@
+@use super::statics::*;
+@use crate::footer;
+
+@(params: &[(&str, usize)])
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Example</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <link rel="stylesheet" type="text/css" href="/static/@style_css.name"/>
+  </head>
+  <body>
+    <main>
+      <h1>Example</h1>
+
+      <p>This is a simple sample page,
+      to be served with warp.
```

Comment:
Maybe change this to "to be served with axum"?  :-)

Reply:
Yeah, I missed that one. I fixed this also for actix and tide in da5776e